### PR TITLE
Move health classes to core package

### PR DESCRIPTION
**Pull request by gkleiman**

### CODE REVIEW

**Comment by aquamatthias** [comment]
LGTM! Thanks. Please rebase and I will merge.

**Comment by gkleiman** [comment]
Rebased

File: src/main/scala/mesosphere/marathon/core/CoreModule.scala [hunk 1]
```diff
@@ -21,17 +22,18 @@ import mesosphere.marathon.core.task.tracker.TaskTrackerModule
   * (as long as we have them).
   */
 trait CoreModule {
```
Comment (gkleiman) [inline src/main/scala/mesosphere/marathon/core/CoreModule.scala]: This looks like a bigger change, but I just sorted the attributes alphabetically and added `healthModule`.

**Comment by aquamatthias** [comment]
Thanks!

**Pull request merged**